File: src/projects/Children/question/src/main/java/com/krm3d/question/persistance/repositories/QuestionRepository.java
package com.krm3d.question.persistance.repositories;

import com.krm3d.question.domain.entities.Question;
import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;

@EnableMongoRepositories
public interface QuestionRepository extends MongoRepository<Question,String> {
}

File: src/projects/Children/ChildMaterializedView/Persistance/Repositories/QuestionRepository.cs
using Application.Services.Repositories;
using Core.Persistance.Repositories.EntityFramework;
using Domain.Entities;
using Persistance.Contexts;

namespace Persistance.Repositories
{
    public class QuestionRepository : EfRepositoryBase<Question, SQLContext>, IQuestionRepository
    {
        public QuestionRepository(SQLContext context) : base(context)
        {
        }
    }

}
